com.dynamo.cr/com.dynamo.cr.bob/scripts/copy.sh

```diff
@@ -79,9 +79,7 @@ cp -v $DYNAMO_HOME/ext/lib/x86_64-win32/wrap_oal.dll lib/x86_64-win32/wrap_oal.d
 
 cp -v $DYNAMO_HOME/ext/bin/x86_64-linux/luajit-32 libexec/x86_64-linux/luajit-32
 cp -v $DYNAMO_HOME/ext/bin/x86_64-linux/luajit-64 libexec/x86_64-linux/luajit-64
-cp -v $DYNAMO_HOME/ext/bin/x86_64-macos/luajit-32 libexec/x86_64-macos/luajit-32
```

Comment:
We are no longer building a luajit-32 capable of producing 32bit bytecode. The `luajit-64` can also produce 32 bit bytecode using the `-W` option.